apps/ledger-live-mobile/src/analytics/segment.ts

```diff
@@ -187,6 +196,7 @@ const extraProperties = async (store: AppStore) => {
     nps,
     optInAnalytics: analyticsEnabled,
     optInPersonalRecommendations: personalizedRecommendationsEnabled,
+    stakingProvidersEnabled: stakingProvidersCount || "flag not loaded",
```

Comment:
If the feature flag is disabled what value do we want to send ?
Same question if the listProvider is an empty array

Reply:
Good q. We'll need to know about that so a semi-descriptive message seems like a better option than `false` or `undefined`